Makefile

```diff
@@ -68,7 +69,7 @@ gen-k8s: $(apis_sources)
 gen-k8s-check: $(apis_sources)
 	./hack/verify-codegen.sh
 
-container-build: container-build-operator container-build-registry
+container-build: container-build-operator container-build-registry build-must-gather
```

Comment:
can we keep naming consistent, so `container-build-must-gather` please?

Reply:
done